src/services/user.service.js

```diff
@@ -0,0 +1,44 @@
+const { data } = require('../data');
+
+const getAll = () => {
+  return data.users;
+};
+
+const getById = (id) => {
+  return data.users.find((user) => user.id === id) || null;
```

Comment:
This function `getById` could potentially return `undefined` if the user id is not found. It would be better to handle this case explicitly and throw an error or return a specific value to indicate that the user was not found.